lib/packages/crypto/src/words.ts

```diff
@@ -0,0 +1,3 @@
+
+    // taken from https://github.com/first20hours/google-10000-english/blob/master/google-10000-english-usa-no-swears-medium.txt
```

Comment:
nice split up...I'd like to refactor the passPhraseGenerator to make the dictionary injectible...